React/Base/RCTConvert.m

```diff
@@ -567,6 +571,25 @@ static BOOL RCTFontIsCondensed(UIFont *font)
   return (symbolicTraits & UIFontDescriptorTraitCondensed) != 0;
 }
 
+NSDictionary *RCTFontVariantMapping = NULL;
```

Comment:
I couldn’t find much information on initialising a static NSDictionary. Is this acceptable?